blackbox/docs

```diff
@@ -1 +0,0 @@
-../docs
```

Comment:
Was it intended to remove this?

Reply:
Hi, no, I accidentally deleted it and restored locally. Trying to check whether it's indeed needed to clone again after enabling dev mode or it's proactive.